packages/modeling/src/operations/modifiers/edges.js

```diff
@@ -0,0 +1,204 @@
+const vec3 = require('../../maths/vec3')
+const line3 = require('../../maths/line3')
+
+const poly3 = require('../../geometries/poly3')
+
+// an edge is an array of two vertices
+const addEdge = (edges, edge, polygon) => {
```

Comment:
Feel like this needs a docblock.  "Add an edge to a polygon, and the polygon to the edge's polygon references. If an equivalent edge is found in the array "edges", then use a reference to that edge instead.".

Reply:
Done